pyproject.toml

```diff
@@ -94,6 +94,18 @@ dev = [
     "wheel",
 ]
 
+dev_precommit = [
+    "black",
+    "check-manifest",
```

Comment:
Is `MANIFEST.in` file modified commonly? If not, this may not be required for usual contributors.

Personal curiosity, can you tell me what does `MANIFEST.in` do? I've few python packages (private for office work) and never needed to use it.